mockwebserver-deprecated/src/main/kotlin/okhttp3/mockwebserver/MockResponse.kt

```diff
@@ -86,62 +87,81 @@ class MockResponse : Cloneable {
 
   @JvmName("-deprecated_getStatus")
   @Deprecated(
-      message = "moved to var",
-      replaceWith = ReplaceWith(expression = "status"),
-      level = DeprecationLevel.ERROR)
+    message = "moved to var",
+    replaceWith = ReplaceWith(expression = "status"),
+    level = DeprecationLevel.ERROR,
+  )
   fun getStatus(): String = status
 
-  fun setStatus(status: String) = apply {
-    this.status = status
-  }
+  fun setStatus(status: String) =
+    apply {
+      this.status = status
+    }
 
   fun setResponseCode(code: Int): MockResponse {
-    val reason = when (code) {
-      in 100..199 -> "Informational"
-      in 200..299 -> "OK"
-      in 300..399 -> "Redirection"
-      in 400..499 -> "Client Error"
-      in 500..599 -> "Server Error"
-      else -> "Mock Response"
-    }
+    val reason =
+      when (code) {
+        in 100..199 -> "Informational"
+        in 200..299 -> "OK"
+        in 300..399 -> "Redirection"
+        in 400..499 -> "Client Error"
+        in 500..599 -> "Server Error"
+        else -> "Mock Response"
+      }
     return apply { status = "HTTP/1.1 $code $reason" }
   }
 
-  fun clearHeaders() = apply {
-    headersBuilder = Headers.Builder()
-  }
+  fun clearHeaders() =
+    apply {
+      headersBuilder = Headers.Builder()
+    }
 
-  fun addHeader(header: String) = apply {
-    headersBuilder.add(header)
-  }
+  fun addHeader(header: String) =
+    apply {
+      headersBuilder.add(header)
+    }
 
-  fun addHeader(name: String, value: Any) = apply {
+  fun addHeader(
+    name: String,
+    value: Any,
+  ) = apply {
     headersBuilder.add(name, value.toString())
   }
```

Comment:
Seems like a bug that it wraps the expression body onto its own line if the function signature is a single line, but if the function signature wraps it does not wrap the expression body.